Gemfile

```diff
@@ -41,7 +39,6 @@ group :test do
   gem 'better_errors'
   gem 'pry', '=0.10.1'
   gem 'pry-rails', '=0.3.3'
-  gem 'rspec-rails', '~> 3.8'
```

Comment:
why are we removing this gem from the test group?

Reply:
its was present in both groups and outside